### PR TITLE
M2-5201: Update readme file

### PR DESCRIPTION
**Pull request by ChaconC**
## Description

[JIRA: M2-5201](https://mindlogger.atlassian.net/browse/M2-5201)

Update README and add LICENSE.md

Pending LICENSE.md review

### CODE REVIEW

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
# Child Mind Institute - MindLogger Admin
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
This repository is used for the Admin Panel of the [MindLogger](https://mindlogger.org/) application stack.
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
Running the app:
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
- Configured [environment variables](#environment-variables):\
```

File: README.md
Comment (farmerpaul) [inline README.md:19]:
```suggestion
- [Backend](https://github.com/ChildMindInstitute/mindlogger-backend-refactor) project running locally or accessible in a test environment
  - If running locally, ensure that `http://localhost:3000` has been added to the BE's `CORS__ALLOW_ORIGINS` environment variable
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
- Run the project using `npm run start`. See [scripts](#available-scripts)
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
See MindLogger's [Knowledge Base article](https://mindlogger.atlassian.net/servicedesk/customer/portal/3/topic/4d9a9ad4-c663-443b-b7fc-be9faf5d9383/article/337444910) to discover the MindLogger application stack's features.
```

File: README.md
Comment (farmerpaul) [inline README.md:46]: I'd suggest a better description of `React-app-rewired` above: `All the benefits of create-react-app without the limitations of "no config"`

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
- `npm run test` - Launches the test runner in the interactive watch mode.
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
| REACT_APP_ENV                      | no | null | Environment to run the app in (`prod` or `staging`)  |
```

File: README.md
Comment (farmerpaul) [inline README.md]:
```suggestion
| REACT_APP_API_DOMAIN               | yes | null | MindLogger Backend API base URL |
```

File: .node-version
Comment (farmerpaul) [inline .node-version]: The README says to use Node `18` or higher – does it make sense to require version `20` here (and in `.nvmrc`)?

**Review by farmerpaul — APPROVED** [review]
Looks great @ChaconC! I just identified a few nits/suggestions. Mostly unimportant things like punctuation or wording adjustments. Happy to preapprove, but would be best to get another set of eyes from ScienceSoft before merging!

File: .node-version
Comment (ChaconC) [reply .node-version]: I put `20` as it's the current LTS version, but personally had issues running with `16` that went away once I moved to 18 or 20...
So this is more of a suggested (`20`) vs required (`18`) 🤔 

Edit: Will go with `18` as that's already in the repo

Comment (farmerpaul) [reply .node-version]: Ah okay sure, fine with me 🙂

File: README.md
Comment (farmerpaul) [inline README.md:5]: Just notice you have another `## Getting Started` heading below. Maybe this should be:
```suggestion
## Application Stack
```
?

File: .node-version
Comment (sultanofcardio) [inline .node-version]: I think they're using node v18, and I added this in #1472 so we could probably remove this file

File: .nvmrc
Comment (sultanofcardio) [inline .nvmrc]: To match the `.node-version` file

```suggestion
v18.19.0
```

https://github.com/ChildMindInstitute/mindlogger-admin/blob/57051d6026285cb4433900db512861886524f04e/.node-version#L1

File: README.md
Comment (ChaconC) [reply README.md:5]: Nice catch! problem with copying sections

**Pull request merged**
